Swiftfin/Views/FontPicker.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2022 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+import UIKit
+
+// https://github.com/SwapnanilDhol/SUIFontPicker
```

Comment:
When the `onFontPick` is removed we can remove this attribution as it's different enough from that source.